app/components/PinListItem.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component, Text, TouchableHighlight, View, StyleSheet, AlertIOS} from 'react-native';
+
+export default class PinListItem extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { pin, deletePin } = this.props;
+    return (
+      <TouchableHighlight 
+        onPress={() => {
+          AlertIOS.alert(
+              'Delete Pin?',
+              null,
+              [{
+                text: 'Cancel',
+                style: 'cancel'
+              },
+              {
+                text: 'OK',
+                onPress: () => {
+                  deletePin(pin)
+                }
+              }]
+            )
```

Comment:
Why did you choose to imbed this code here instead of breaking it out into a fn?